OJP/OJP_JourneySupport.xsd

```diff
@@ -211,46 +211,22 @@
 			<xs:documentation>Vehicle journey that runs at a specific date.</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
-			<xs:element ref="OperatingDayRef">
+			<xs:element name="ConventionalModeOfOperation" type="ConventionalModesOfOperationEnumeration" minOccurs="0" default="scheduled">
 				<xs:annotation>
-					<xs:documentation>Reference to an Operating Day.</xs:documentation>
+					<xs:documentation>MODE OF OPERATION for scheduled and flexible services (NeTEx).</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element ref="siri:VehicleRef" minOccurs="0">
-				<xs:annotation>
-					<xs:documentation>Contains the Vehicle reference of the vehicle. In Transmodel this may be the VEHICLE Id.</xs:documentation>
-				</xs:annotation>
-			</xs:element>
-			<xs:group ref="ScheduledJourneyGroup"/>
-		</xs:sequence>
-	</xs:group>
-	<xs:group name="PrivateServiceGroup">
-		<xs:annotation>
-			<xs:documentation>Specialized service operated by a Private Mode, e.g. a school bus, corporate shuttle, or other transportation for a (usually) closed community.</xs:documentation>
-		</xs:annotation>
-		<xs:sequence>
-			<xs:element ref="JourneyRef" minOccurs="0"/>
-			<xs:element name="PrivateMode" type="PrivateModesEnumeration">
+			<xs:element ref="OperatingDayRef">
 				<xs:annotation>
-					<xs:documentation>[a category of MODE in TMv6] MODEs offered by private individuals.</xs:documentation>
+					<xs:documentation>Reference to an Operating Day.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
 			<xs:element ref="siri:VehicleRef" minOccurs="0">
 				<xs:annotation>
 					<xs:documentation>Contains the Vehicle reference of the vehicle. In Transmodel this may be the VEHICLE Id.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element name="Description" type="siri:NaturalLanguageStringStructure" minOccurs="0">
-				<xs:annotation>
-					<xs:documentation>Description of the private service.</xs:documentation>
-				</xs:annotation>
-			</xs:element>
-			<xs:element name="OrganisationRef" type="siri:OperatorRefStructure" minOccurs="0"/>
-			<xs:element name="InfoUrl" type="WebLinkStructure" minOccurs="0">
-				<xs:annotation>
-					<xs:documentation>Link to web page providing more details on service.</xs:documentation>
-				</xs:annotation>
-			</xs:element>
+			<xs:group ref="ScheduledJourneyGroup"/>
```

Comment:
`ScheduledJouneyGroup` already contains an optional `siri:VehicleRef` - remove it above (currently line 238/224)?